src/test/ui/consts/unwind-abort.rs

```diff
@@ -0,0 +1,17 @@
+// check-pass
+
+#![feature(unwind_attributes, const_panic)]
+
+// `#[unwind(aborts)]` is okay for a `const fn`. We don't unwind in const-eval anyways.
```

Comment:
Why did you add two tests? Our test suite is already huge.^^

Reply:
One is check-pass, the other fails to compile.

Reply:
I can see that, I just do not understand their respective purpose. Specifically, "consts/unwind-abort" does not even run any CTFE.

Reply:
:shrug: I'm happy with the test coverage in this PR.

Reply:
I am trying to understand what `unwind-abort.rs` tests that the other test does not already cover.

My concern is not too little coverage; my concern is unnecessarily growing the number of UI tests.

Reply:
I try to split up passing and failing tests when possible. One is solely a test of const-checking and involves no const-eval, the other is a sanity check to ensure that the CTFE engine doesn't do anything weird when evaluating something with `#[unwind(aborts)]`.

Reply:
I think this unnecessarily leads to an excessive number of tests in our UI test suite that is already so big that it becomes hard to run the entire thing, but whatever.

Could you add comments to these tests explaining what they are meant to test?